frontendservice/src/components/statusUpdate.vue

```diff
@@ -0,0 +1,34 @@
+/* Copyright 2020 Google LLC Licensed under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance with the License. You
+may obtain a copy of the License at https://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License. */
+
+<template>
+  <div>
+    <div class="notification is-danger">
+      <button class="delete" onclick="this.parentElement.style.display='none'"></button>
+      <strong>Your token has been deleted by the shop owner!</strong>
```

Comment:
We decided to remove this from the PR, right?

Reply:
I have included this component in this pr but it is not dependent on the deletion of token in backend in this pr, I resolved the fluctuation issue and included that logic in the shopOwner-DeleteFrontend pr.